File: .sdk/tm/ts/test/exists.test.ts

import { test, describe } from 'node:test'
import { equal } from 'node:assert'


import { OpenDataHubSDK } from '..'


describe('exists', async () => {

  test('test-mode', async () => {
    const testsdk = await OpenDataHubSDK.test()
    equal(null !== testsdk, true)
  })

})

File: rb/test/exists_test.rb
# OpenDataHub SDK exists test

require "minitest/autorun"
require_relative "../OpenDataHub_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = OpenDataHubSDK.test(nil, nil)
    assert !testsdk.nil?
  end
end
